mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -209,11 +185,15 @@ def wiki_query(data_csv, user_agent):
         response.raise_for_status()
         data = response.json()
 
-        wiki_movie_ids.append(wiki_feature_info(data, "item"))
-        wiki_genres.append(wiki_feature_info(data, "genreLabel"))
-        wiki_directors.append(wiki_feature_info(data, "directorLabel"))
+        wiki_data_list.append(
```

Comment:
Very important, you also need to update the `process_data` function, where this is called, and have it handle the data in the new format.

Reply:
@JamesKohlsRepo , your IDE is a power tool. Take advantage of it! Practice makes perfect.

Renaming is a pretty common refactor, and not just for CamelVersus_snake_case minutiae. Commonly we will "guess" at what something means, write code, then realize it means something a little different so we should rename to reflect that. Begin by putting the cursor on the identifier we wish to change.

In Pycharm, right-click to get a context menu, choose Refactor => Rename, adjust to the new spelling of the identifier, hit RETURN.

In VSCode, right-click, choose Rename, type the name, hit RETURN.

The IDE will then chase through the codebase to adjust the definition plus all calling sites, including `import` statements. Use `$ git diff` to verify that things went as planned, then commit the edit.